### PR TITLE
The Golden Zipper

### PR DESCRIPTION
**Pull request by DVDPlayerOfDiscordFame**
Adds a funny impact effect asset to play with, as well as a fun toy for admins and gamblers.

**Changelog**

:cl:
- add: A fun toy that can be found in a grand lottery crate.
- add: A Very Fun Toy that admins can play with and if you are astronomically lucky can find in a grand lottery crate.

### CODE REVIEW

File: Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Battery/goldzipper.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - type: PointLight
+    radius: 1.2
+    energy: 1.34
+    castShadows: false
+    color: "#9b68ff"
+    
+- type: entity
+  name: golden zipper
+  parent: WeaponGZipper
+  id: WeaponFGZipper
```
Comment (Darkmajia) [inline Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Battery/goldzipper.yml:42]: in a similar vein, "WeaponGoldZipperFake" would be best for clarity, and additions to a base entity's id tend to go at the end. a suffix for the spawn menu would also be nice for admins/mappers to be able to tell them apart

Comment (DVDPlayerOfDiscordFame) [reply Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Battery/goldzipper.yml]: Great point! I've changed both to that, thank you.

File: Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Battery/goldzipper.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    radius: 1.2
+    energy: 1.34
+    castShadows: false
+    color: "#9b68ff"
+    
+- type: entity
+  name: golden zipper
+  parent: WeaponGoldZipper
+  id: WeaponGoldZipperFake
+  description: Legends tell of a gun that can kill anything with a single shot. This one, however, seems to be made of cheap plastic.
```
Comment (Darkmajia) [inline Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Battery/goldzipper.yml:43]: if you can add "suffix: Fake" in here to make them easily differentiated in the spawn menu i will merge this asap

**Review by formlessnameless — APPROVED** [review]

**Review by Darkmajia — APPROVED** [review]

**Comment by Darkmajia** [comment]
pr so nice we approve it twice

**Pull request merged**